nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -360,12 +369,12 @@ def connect_thread_func(self):
             # Close sftp connection and release ssh connection lock
             sftp_client.close()
 
-            # Delete argument file (no longer needed)
-            os.remove(self.local_data_filepath)
-
         # Invoke a shell in the ssh client
         ssh_channel = self.ssh_client.invoke_shell()
 
+        # Wait for the SSH shell to initialize
+        time.sleep(0.1)
```

Comment:
Not that I've found. And I only encountered this error randomly on the DE1. Since we are going to do an SSH rework, I'm inclined to leave this simple hack in.